### PR TITLE
mark svelte-kit package as experimental

### PR DESCRIPTION
**Pull request by Rich-Harris**
At the last maintainer's meeting we decided to mark `svelte-kit package` as experimental, as there are certain limitations we want to address that could involve breaking changes (mainly that it doesn't work great with monorepos).

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: documentation/docs/11-packaging.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: Packaging
 ---
 
+> `svelte-kit package` is experimental, and may undergo breaking changes.
```
Comment (ignatiusmb) [inline documentation/docs/11-packaging.md:5]:
```suggestion
> `svelte-kit package` is currently experimental and is not subject to Semantic Versioning rules. Non-backward compatible changes or removal may occur in any future release.
```

File: documentation/docs/12-cli.md [hunk 2]
```diff
@@ -40,4 +40,6 @@ Like `svelte-kit dev`, it accepts the following options:
 
 ### svelte-kit package
 
+> `svelte-kit package` is experimental, and may undergo breaking changes.
```
Comment (ignatiusmb) [inline documentation/docs/12-cli.md:43]:
```suggestion
> `svelte-kit package` is currently experimental and is not subject to Semantic Versioning rules. Non-backward compatible changes or removal may occur in any future release.
```

**Review by ignatiusmb — APPROVED** [review]
Should be good enough, but I think a bit more explanation, instead of just "may undergo breaking changes" will not scare away too much potential users from actually trying this out.

File: documentation/docs/11-packaging.md (hunk 1 above)
Comment (Rich-Harris) [inline documentation/docs/11-packaging.md:5]:
```suggestion
> `svelte-kit package` is currently experimental and is not subject to Semantic Versioning rules. Non-backward compatible changes may occur in any future release.
```

**Comment by Rich-Harris** [comment]
I removed the 'or removal', since it sounds scary and we don't expect that to happen (but if it _does_ happen, it's covered by 'breaking changes')

**Pull request merged**